pkg/controller/sinks/sinks.go

```diff
@@ -41,18 +41,20 @@ func GetSinkURI(ctx context.Context, c client.Client, sink *corev1.ObjectReferen
 		return "", err
 	}
 
+	objIdentifier := fmt.Sprintf("\"%s/%s\" (%s.%s)", u.GetNamespace(), u.GetName(), u.GetAPIVersion(), u.GetKind())
```

Comment:
Optional: You can use backticks here too to avoid having to escape inner quotes, if that's your bag.

Reply:
I'm happy to use backticks in tests, where I do think it makes things more readable, but I'm inclined to use double-quotes and escaping here so that readers don't need to remember if there's anything special about <code>`</code>.

Reply:
instead of manually creating the u.GetAPIVersion, u.GetKind, it might be better to just use:
u.GroupVersionKind()?
Or, it's really no different from:
sink.GeroupVersionKind so it's got the standard k8s formatted string.

Reply:
Hey, I hadn't seen `GroupVersionKind()`. Switched, but it makes the output slightly more verbose.